### DISCUSSION TITLE
location of trivy-offline.db.tgz not available

### DISCUSSION THREAD

**Discussion by santoshkarp3** [aquasecurity/trivy#4808] [Q&A]
### Description

Location: https://github.com/aquasecurity/trivy-db/releases/download/v1-end-of-support/trivy-offline.db.tgz [following]
--2023-07-12 03:08:38--  https://github.com/aquasecurity/trivy-db/releases/download/v1-end-of-support/trivy-offline.db.tgz
Reusing existing connection to github.com:443.
Proxy request sent, awaiting response... 404 Not Found
2023-07-12 03:08:38 ERROR 404: Not Found.

### Link

https://aquasecurity.github.io/trivy/v0.27.1/docs/advanced/air-gap/

### Suggestions

Please correct and provide the correct link to download offline trivy-db file

**Comment by simar7** [comment]
You seem to be using a very old version of Trivy. Please use the latest release here: https://aquasecurity.github.io/trivy/v0.43/docs/advanced/air-gap/

**Reply by santoshkarp3** [reply]
Thanks for reply ,
I am getting error in starboard for trivy-db 
2023-06-30T12:42:12.371Z  INFO             DB Repository: ghcr.io/aquasecurity/trivy-db
2023-06-30T12:42:12.371Z  INFO             Downloading DB...
2023-06-30T12:42:12.379Z  FATAL            DB error: failed to download vulnerability DB: OCI artifact error: OCI artifact error: OCI repository error: Get "https://ghcr.io/v2/": unexpected EOF


its airgap environment , i want to try with downloading trivy-db offline and supply it to starboard 

Can you suggest please

**Comment by santoshkarp3** [comment]
I am using starboard operator latest version , issue is unable to downloads trivy-db file in airgap environment
